components/Hackathons.js

```diff
@@ -6,20 +6,20 @@ import { Body, Title2 } from './Typography';
 import Button from './Button';
 
 const HackCampData = {
-  imgSrc: '/assets/HackCamp2023.png',
+  imgSrc: '/assets/hc2023-banner.png',
   link: 'https://hackcamp.nwplus.io',
   date: 'Nov 9 - 10',
   registrationOpenDate: "Oct 6",
   open: false,
 };
 const nwHacksData = {
-  imgSrc: '/assets/nwHacks2024.png',
+  imgSrc: '/assets/nwHacks2024-banner.png',
```

Comment:
```suggestion
  imgSrc: '/assets/nwhacks2024-banner.png',
```
![image](https://github.com/user-attachments/assets/85680ceb-d7bc-4c9d-8f09-ae4a96df24bd)

Reply:
fixed! ty ty